liblepton/src/component_object.c

```diff
@@ -1514,3 +1514,78 @@ lepton_component_object_set_missing (const LeptonObject *object,
 
   object->component->missing = missing;
 }
+
+
+/*! \brief Embed a component object into its schematic.
+ *  \par Function Description
+ *  This function embeds a component object into its schematic.
+ *  The object is just marked as embedded.
+ *
+ *  \param object The #LeptonObject to embed
+ */
+void
+lepton_component_object_embed (LeptonObject *object)
+{
+  LeptonPage *page;
+
+  g_return_if_fail (lepton_object_is_component (object));
+
+  page = lepton_object_get_page (object);
```

Comment:
Shouldn't we check here that `page != NULL`?

Reply:
Thank you for the catch!  Apparently, I've confused the func with `lepton_object_page_set_changed()` which does the check inside it.  I'll add the fix soon.